pkg/apis/networking/types.go

```diff
@@ -349,18 +349,27 @@ type IngressStatus struct {
 // host match, then routed to the backend associated with the matching
 // IngressRuleValue.
 type IngressRule struct {
-	// Host is the fully qualified domain name of a network host, as defined
-	// by RFC 3986. Note the following deviations from the "host" part of the
-	// URI as defined in the RFC:
-	// 1. IPs are not allowed. Currently an IngressRuleValue can only apply to the
-	//	  IP in the Spec of the parent Ingress.
+	// Note the following deviations from the "host" part of the
```

Comment:
nit: keep the first sentence of the godoc beginning with the field name to match conventions